torch_geometric/nn/conv/rgcn_conv.py

```diff
@@ -102,6 +105,9 @@ def __init__(
         bias: bool = True,
         **kwargs,
     ):
+        warnings.warn('This RGCNConv can use dynamic shapes. If your device '
+                      'doesn\'t support dynamic shapes, use FastRGCNConv.')
+
```

Comment:
Lets not raise a warning, if its documented.

Reply:
Done.